pkg/client/client.go

```diff
@@ -63,3 +71,63 @@ func (n Node) Join(ctx context.Context, topic string) *Topic {
 
 	return t
 }
+
+func (n Node) Ls(ctx context.Context, path []string) (cluster.AnchorIterator, error) {
```

Comment:
`Ls` shouldn't take a path argument.  It should just return the immediate children of the Anchor.  Since `Node` is exactly the root anchor, this amounts to the `if len(path) == 0` block.

To list the children of other anchors, users will walk to them first.